test/test_adjoint.py

```diff
@@ -338,14 +343,21 @@ def g(t, y, args):
         _, k1 = args
         return k1 * y
 
+    def g_lx(t, y, args):
+        _, k1 = args
+        return jtu.tree_map(lx.DiagonalLinearOperator, k1 * y)
```

Comment:
Why the `tree_map`?

Reply:
Leftover from a older version I made probably, removed